### ISSUE TITLE
Remove the need to tag hyperledger/fabric-baseimage:latest

### ISSUE THREAD

**Issue by mhart** [fabric-composer/fabric-composer#289]
I'm not really sure behind the reasoning of this, but all of the samples have lines similar to this:

```
docker pull hyperledger/fabric-baseimage:x86_64-0.1.0
docker tag hyperledger/fabric-baseimage:x86_64-0.1.0 hyperledger/fabric-baseimage:latest
```

And indeed I wasn't able to get composer talking to fabric without doing this – it would just timeout when deploying a network.

Would be great to remove this requirement – it's very non-obvious to newbies

**Comment by sstone1** [comment]
@mhart this is a Hyperledger Fabric (HLF) v0.6 restriction, which will hopefully be removed when HLF v1.0 is released. Unfortunately, there is nothing that we can do from the Fabric Composer side to remove this requirement.

The problem is caused by the deploy process using the HLF client SDK (hfc) creating a Dockerfile for the chaincode that contains the line `FROM hyperledger/fabric-baseimage`. No tag is specified, because the HFC client SDK does not know which version should be used, as it does not know the version or architecture of the peer(s), and so the build of the chaincode Docker container defaults to the latest tag.

However, the HLF team are not publishing a latest tag to Docker Hub. This causes the deploy to fail with an error stating that Docker could not find/pull an image for `hyperledger/fabric-baseimage:latest`. You will be able to see this error if you run `docker logs` against the Docker container ID of the peer.

The relevant HLF JIRA items are:
https://jira.hyperledger.org/browse/FAB-300
https://jira.hyperledger.org/browse/FAB-678

**Comment by mhart** [comment]
Thanks for the explanation @sstone1 – looking forward to this being fixed in HLF v1.0
